website/pages/docs/plugins/sources/overview.mdx

```diff
@@ -24,6 +24,7 @@ Official source plugins follow [release stages](#source-plugin-release-stages).
 | [Okta][okta] | {getLatestVersion("source", "okta")}        | [Changelog][Okta-Changelog]         | [Tables][Okta-Tables]          | Preview |
 | [Terraform][terraform] | {getLatestVersion("source", "terraform")}         | [Changelog][Terraform-Changelog]    | [Tables][Terraform-Tables]     | Preview |
 | [Cloudflare][cloudflare] | {getLatestVersion("source", "cloudflare")}         | [Changelog][Cloudflare-Changelog]   | [Tables][Cloudflare-Tables]    | Preview |
+| [Gandi][gandi] | {getLatestVersion("source", "gandi")}         | [Changelog][Gandi-Changelog]   | [Tables][Gandi-Tables]    | Preview |
```

Comment:
```suggestion
| [Gandi][gandi] | {getLatestVersion("source", `gandi`)}         | [Changelog][Gandi-Changelog]   | [Tables][Gandi-Tables]    | Preview |
```

Reply:
@erezrokah why though, when the other source plugins have the double quotes and not backticks?

Reply:
I'm not sure actually. It's a bit tricky to lint MDX files so I might disable it as they have code + text.
The backticks makes our linter thinks it's code and not text so it doesn't lint it (hacky workaround)